addons/website_mail/views/website_mail_templates.xml

```diff
@@ -20,77 +20,26 @@
-                <h3>Leave a comment</h3>
-                <p>You must be <a t-attf-href="/web/login?redirect=#{quote_plus(request.httprequest.url + '#discussion')}">logged in</a> to post a comment.</p>
-            </section>
-            <div class="clearfix"/>
-            <ul class="media-list o_website_comments">
-                <t t-set="messages" t-value="object.message_ids"/>
-                <t groups="base.group_portal,base.group_public">
-                    <t t-set="messages" t-value="object.message_ids.filtered(lambda m: m.subtype_id.internal == False)"/>
-                </t>
-                <t t-foreach="messages" t-as="message">
-                    <t t-if="message.website_published or website.env.user.sudo(website.env.user.id).has_group('website.group_website_publisher')">
-                        <li class="media" t-if="message.message_type != 'notification'" t-att-id="'message-%s' % message.id">
-                            <div class="media-body">
-                                <img class="media-object pull-left img-circle" t-attf-src="/mail/#{message.model}/#{message.res_id}/avatar/#{message.author_id.id}"
-                                    style="width: 50px; margin-right: 10px;"/>
-                            <div class="media-body">
-                                <h5 class="media-heading">
-                                    <span t-field="message.author_id"/> <small>on <span t-field="message.date"/></small>
-                                </h5>
-                                <t t-set="object" t-value="message"/>
-                                <t t-call="website.publish_short"/>
-                                <div t-field="message.body"/>
-                                    <div class="col-md-2 col-sm-3 text-center" t-foreach='message.attachment_ids' t-as='attachment'>
-                                        <a t-attf-href="/web/content/#{attachment.id}?download=true" target="_blank">
-                                            <div class='oe_attachment_embedded o_image' t-att-title="attachment.name" t-att-data-mimetype="attachment.mimetype" t-attf-data-src="/web/image/#{attachment.id}/100x80"/>
-                                            <div class='oe_attachment_name'><t t-esc='attachment.name' /></div>
-                                        </a>
-                                    </div>
-                            </div>
-                            </div>
-                        </li>
-                    </t>
-                </t>
-            </ul>
-        </section>
+        <div id="discussion" class="hidden-print o_website_mail_thread"
+            t-att-data-token="token" t-att-public-user="website.env.user == request.website.user_id" t-att-data-res_model="object._name" t-att-data-res_id="object.id" t-att-data-token_field="token_field" t-att-data-pager_step="message_per_page or 10" t-att-data-allow_composer="'0' if disable_composer else '1'">
```

Comment:
Setting  `t-att-token=="object[token_field]"` will crash if token_field is not set. And since we usually use the template without the token mecanism, it should be better like this ;)